pkg/apis/migration/v1alpha1/migplan_types.go

```diff
@@ -211,10 +212,14 @@ const (
 func (r *MigPlan) BuildRegistrySecret(client k8sclient.Client, storage *MigStorage) (*kapi.Secret, error) {
 	labels := r.GetCorrelationLabels()
 	labels[MigrationRegistryLabel] = string(r.UID)
+	strippedName, err1 := toDnsLabel(r.GetName())
```

Comment:
NIT: Why use `err1`? Could just use `err` and reuse the var down below for `UpdateRegistrySecret`.

Reply:
This was a workaround for an unexplained error that the compiler was giving me. I hadn't even noticed the redeclaration below because the compiler was complaining about something unrelated instead of this. I've updated the PR, as changing the second declaration removed the problem.